### PR TITLE
Swipe component

### PR DESCRIPTION
**Pull request by froskie**
## What this PR do
- Adds the Swipe component
image

## Where should the reviewer start:
- Diffs

## Unit and/or functional tests:
- 100% on the swipe folder

### CODE REVIEW

File: components/swipe/swipeMouse.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Touch = ({ children, onEnd, onMove, onStart, ...rest }) => (
```
Comment (asci) [inline components/swipe/swipeMouse.js]: Move?

Comment (froskie) [inline components/swipe/swipeMouse.js]: ...?

Comment (asci) [inline components/swipe/swipeMouse.js]: The name — for touch events: Touch, I guess for mouse events should be Move (or so)?

Comment (froskie) [inline components/swipe/swipeMouse.js]: Start - MouseDown/TouchStart
Move - MouseMove/TouchMove
End - MouseUp/TouchEnd

**Pull request closed without merging**